jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -857,47 +667,3874 @@ public String(char[] data) {
+	 *
+	 * @param data
+	 *			  the array of characters
+	 * @return the String
+	 *
+	 * @throws NullPointerException
+	 *				when data is null
+	 */
+	public static String valueOf(char[] data) {
+		return new String(data, 0, data.length);
+	}
+
+	/**
+	 * Returns a String containing the specified characters in the character array. Modifying the character array after creating the String has no
+	 * effect on the String.
+	 *
+	 * @param data
+	 *			  the array of characters
+	 * @param start
+	 *			  the starting offset in the character array
+	 * @param length
+	 *			  the number of characters to use
+	 * @return the String
+	 *
+	 * @throws IndexOutOfBoundsException
+	 *				when {@code length < 0, start < 0} or {@code start + length > data.length}
+	 * @throws NullPointerException
+	 *				when data is null
+	 */
+	public static String valueOf(char[] data, int start, int length) {
+		return new String(data, start, length);
+	}
+
+	/**
+	 * Converts the specified character to its string representation.
+	 *
+	 * @param value
+	 *			  the character
+	 * @return the character converted to a string
+	 */
+	public static String valueOf(char value) {
+		String string;
+
+		if (value <= 255) {
+		   if (enableCompression) {
+		       string = new String(compressedAsciiTable[value], 0, 1, true);
+		   } else {
+		       string = new String(decompressedAsciiTable[value], 0, 1, false);
+		   }
+		} else {
+			byte[] buffer = new byte[2];
+			
+			helpers.putCharInArrayByIndex(buffer, 0, value);
+			
+		   string = new String(buffer, 0, 1, false);
+		}
+
+		return string;
+	}
+
+	/**
+	 * Converts the specified double to its string representation.
+	 *
+	 * @param value
+	 *			  the double
+	 * @return the double converted to a string
+	 */
+	public static String valueOf(double value) {
+		return Double.toString(value);
+	}
+
+	/**
+	 * Converts the specified float to its string representation.
+	 *
+	 * @param value
+	 *			  the float
+	 * @return the float converted to a string
+	 */
+	public static String valueOf(float value) {
+		return Float.toString(value);
+	}
+
+	/**
+	 * Converts the specified integer to its string representation.
+	 *
+	 * @param value
+	 *			  the integer
+	 * @return the integer converted to a string
+	 */
+	public static String valueOf(int value) {
+		return Integer.toString(value);
+	}
+
+	/**
+	 * Converts the specified long to its string representation.
+	 *
+	 * @param value
+	 *			  the long
+	 * @return the long converted to a string
+	 */
+	public static String valueOf(long value) {
+		return Long.toString(value);
+	}
+
+	/**
+	 * Converts the specified object to its string representation. If the object is null answer the string {@code "null"}, otherwise use
+	 * {@code toString()} to get the string representation.
+	 *
+	 * @param value
+	 *			  the object
+	 * @return the object converted to a string
+	 */
+	public static String valueOf(Object value) {
+		return value != null ? value.toString() : "null"
```

Comment:
To include only for Java 11, put `/*[IF Java11]*/` before the method, and `/*[ENDIF]*/` afterwards. You can find other examples of this in this file. These are java preprocessor commands, this source is preprocessed to the correct level before being used for a Java version.

Reply:
Ha, I see its already included in a Java 11 block. Missed that at first.